scipy/stats/_stats_py.py

```diff
@@ -7617,11 +7628,23 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', method='auto'):
 
     Returns
     -------
-    statistic : float
-        KS test statistic, either D, D+ or D- (depending on the value
-        of 'alternative')
-    pvalue : float
-        One-tailed or two-tailed p-value.
+    res: KstestResult
+        An object containing attributes:
+
+        statistic : float
+            KS test statistic, either D, D+ or D- (depending on the value
+            of 'alternative')
+        pvalue : float
+            One-tailed or two-tailed p-value.
+        statistic_location: float
+            Value of `x` corresponding with the KS statistic; i.e., the 
+            distance between the hypothesized cumulative distribution function
+            and the empirical distribution function is greatest at this
+            location.
```

Comment:
Is this what you wanted `statistic_location` for `ks_1samp` to be?

It's not that. Instead, it is the CDF at which the statistic is measured.

Reply:
Oops, that explains why the ks_2samp tests are fine but I couldn't get the one sample tests to work... Will fix tonight, thank you